bigbluebutton-html5/imports/utils/statuses.js

```diff
@@ -0,0 +1,3 @@
+const EMOJI_STATUSES = ['time', 'hand', 'undecided', 'confused', 'sad', 'happy', 'clear-status'];
```

Comment:
These are not the same strings like the rest of BBB. Please use the following ones: [none, raiseHand, happy, neutral, sad, confused, away, thumbsUp, thumbsDown, applause]  and test against the Flash client. The html5 client user statuses should display fine for a flash client user and vice versa.